site/pages/search-results-contextual-en.hbs

```diff
@@ -0,0 +1,104 @@
+---
+{
+	"title": "Search results",
+	"language": "en",
+	"altLangPrefix": "search",
+	"pageType": "search",
+	"pageclass": "page-type-search",
+	"breadcrumb": [
+		{ "title": "Home", "link": "https://www.canada.ca/en.html" }
+	],
+	"dateModified": "2018-12-21",
+	"share": "false",
+	"deptfeature": false
+}
+---
+<!--Note: visual example with styles applied: https://gc-proto.github.io/student-loans/scenarioA/services/search-results-contextual-visual.html-->
+<!--Note 2: this template should only be used for contextual search results from a single institution-->
+<!--Note 3: until radio buttons pattern works, please simply include 'All government of Canada websites' link in the well, under the search box-->
+<div class="col-md-12">
```

Comment:
Remove this column ```div.col-md-12```, it's a bootlint issue and it's useless in this design.

Reply:
Okay